cross/java-11-openjdk/Makefile

```
@@ -20,6 +20,11 @@ BUILD_DEPENDS += cross/fontconfig
 BUILD_DEPENDS += cross/libX11 cross/libXext cross/libXrender cross/libXrandr cross/libXtst cross/libXt
 # libffi is required by the Zero version of Hotspot.
 BUILD_DEPENDS += cross/libffi
+BUILD_DEPENDS += cross/zlib
+BUILD_DEPENDS += cross/libpng
+BUILD_DEPENDS += cross/libjpeg
+BUILD_DEPENDS += cross/giflib
+BUILD_DEPENDS += cross/lcms2
```

now this **MUST** be DEPENDS and not BUILD_DEPENDS as you want to have to include the libraries in the package.
```suggestion
DEPENDS += cross/zlib
DEPENDS += cross/libpng
DEPENDS += cross/libjpeg
DEPENDS += cross/giflib
DEPENDS += cross/lcms2
```

---

Well, yes and no.

The files are already on the DS, installed by Synology, So its only extra security to have them installed in the packages ./lib dir.
Ofc. when/if Synology update the system libs, the Community Builds "might" not work if not updated. If there are no major API updates in ie. zlib then an updated system zlib should work, though compiled towards an older zlib.

Your choice - I would not be so worried.